lib/steps/setupRedirects.js

```diff
@@ -50,6 +56,12 @@ const setupRedirects = () => {
     });
   });
 
+  // This redirect takes care of this issue: https://github.com/netlify/next-on-netlify/issues/43
+  // where the page chunk for a root level catch-all is served incorrectly to the client.
+  // Add last because it's least specific.
+  if (nextRedirects.length >= 1 && hasCatchAllRoute)
+    redirects.push("/_next/*  /_next/static/*  200");
```

Comment:
This is an invalid redirect? /_next/ -> /_next/static ? This would redirects /_next/static/xyz to /_next/static/static/xyz

Alos, I believe the star must be `:splat` on the target: https://docs.netlify.com/routing/redirects/redirect-options/#splats